apps/obsidian/src/components/RelationshipSettings.tsx

```diff
@@ -0,0 +1,296 @@
+import { useState } from "react";
+import {
+  DiscourseRelation,
+  DiscourseNode,
+  DiscourseRelationType,
+} from "../types";
```

Comment:
Let's use the `~` convention instead of `..` for imports to reference the src directory consistently.